### PR TITLE
404 Page Tutorial Fix

### PR DESCRIPTION
**Pull request by bszyman**
## Description

The "Implement Custom Error Pages" tutorial instructs users to create a doc type with the name "404."

`3. Name the Document Type 404.`

This is an invalid name, and the CMS will not allow the user to create/save the new doc type.

umbraco-404-not-allowed

I reworked the existing content to instruct users to name the doc type "404 Page" and manually set the alias to "404Page."

This should tighten up the tutorial and add a little clarity with what the CMS does/doesn't allow.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

## Product & version (if relevant)

v14

## Deadline (if relevant)

N/A

### CODE REVIEW

File: 14/umbraco-cms/tutorials/custom-error-page.md [hunk 1]
```diff
@@ -36,18 +36,19 @@ This method will use a 404 page created via the backoffice.
 
 1. Navigate to the **Settings** section.
 2. Create a new "_Document Type with Template_".
-3. Name the Document Type **404**.
+3. Name the Document Type **404 Page**.
+   1. Note: By default, Umbraco will generate the alias for this DocType as simply "Page," omitting the "404." You may want to manually adjust the alias to "404Page."
```
Comment (sofietoft) [inline 14/umbraco-cms/tutorials/custom-error-page.md:40]: How about adding a new step for actually updating the alias instead?
Like, "`4. Change the alias to **Page404**`" or something similar.

What do you think?

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @bszyman ! 💪 
Super to get this updated 😄 

I've added a single suggestion.

File: 14/umbraco-cms/tutorials/custom-error-page.md (hunk 1 above)
Comment (bszyman) [reply 14/umbraco-cms/tutorials/custom-error-page.md]: What if I told you that I completely agree, on re-reading this the second time though? :)

I added a step 4 and updated the rest of the sequence. I also left a note on step 4 to give the "why." It feels awkward maybe, but I think it's important to tell people why they need to do this.

Submitted a change, LMK what you think!

File: 14/umbraco-cms/tutorials/custom-error-page.md [hunk 2]
```diff
@@ -36,19 +36,19 @@ This method will use a 404 page created via the backoffice.
 
 1. Navigate to the **Settings** section.
 2. Create a new "_Document Type with Template_".
-3. Name the Document Type **404 Page**.
-   1. Note: By default, Umbraco will generate the alias for this DocType as simply "Page," omitting the "404." You may want to manually adjust the alias to "404Page."
-4. \[Optional] Add properties on the Document Type.
+3. Name the Document Type **404**.
+4. Before saving the Document Type, adjust the alias to "Page404." Umbraco will not allow you to create a Document Type with a numeric alias.
```
Comment (sofietoft) [inline 14/umbraco-cms/tutorials/custom-error-page.md:40]: Just a tiny rewrite, to start the step with the _action_ word, "adjust" 😄 

```suggestion
4. Adjust the alias to "Page404" before saving the Document Type. Umbraco will not allow you to create a Document Type with a numeric alias.
```

**Review by sofietoft — COMMENTED** [review]
Looks great @bszyman !
I agree that it is a good idea to explain why the alias needs to be updated 💪 

I've made a tiny suggestion - let me know if something doesn't make sense!

File: 14/umbraco-cms/tutorials/custom-error-page.md (hunk 2 above)
Comment (bszyman) [reply 14/umbraco-cms/tutorials/custom-error-page.md]: @sofietoft - I love it!

**Review by sofietoft — APPROVED** [review]
Thanks again @bszyman ! 
Let's get this one merged 😁

**Comment by nul800sebastiaan** [comment]
@bszyman Good day! You still have a few days to let us know what you want for Hacktoberfest 🎁, we'd love it if it was an active choice!
If you just want us to plant trees 🌳 then let me know or give the form a quick fill (only 2 fields to fill in for trees).
https://feedback.umbraco.com/umbraco-hacktoberfest-2024/

Thanks!

**Pull request merged**